data/ecosystem/adopters.yaml

```diff
@@ -95,6 +95,11 @@
   reference: /blog/2023/end-user-q-and-a-02/
   referenceTitle: blog post
   contact: ''
+- name: VTEX
+  url: https://www.vtex.com/
+  components: []
```

Comment:
```suggestion
  components: [Collector]
```

based on the slides I assume the collector is in use.

any other components you are willing to share?

Reply:
yes, thank you!